charts/opentelemetry-demo-datadog/values.yaml

```diff
@@ -2,9 +2,10 @@ deployMode: "deployment"
 useOperator: false
 
 image:
-  repository:  otel/demo
-  otelcolversion: 0.59.0
+  repository:  dineshgurumurthydd/oteldemo
```

Comment:
We should have this on our otel org asap. Can you talk to #eee?

Reply:
Will do . It is part of our work for ci/cd we want to tackle this quarter .

Reply:
Surely we don't need a whole quarter to add a Docker repository to our org.

Reply:
I meant this work is part of CI/CD we want for our repos. I don't think we should block this PR for that . It is not gonna in next 2 weeks .

Reply:
Agreed. It's a non-blocker. I have approved the PR. It was just a comment.